.github/workflows/release_pre_built/action.yml

```diff
@@ -34,8 +32,6 @@ runs:
         export ELIXIR_ZIP=$PWD/elixir-otp-${{ inputs.otp }}.zip
         (cd lib/elixir/scripts/windows_installer && ./build.sh)
         mv lib/elixir/scripts/windows_installer/tmp/elixir-otp-${{ inputs.otp }}.exe .
```

Comment:
For anyone interested, Azure trusted singing needs to happen after this line, I'll try to submit fix today or tomorrow.